Assets/HoloToolkit/Input/Scripts/ControllerVisualizer.cs

```diff
@@ -45,6 +46,10 @@ public void OnSourceDetected(SourceStateEventData eventData)
                 {
                     GameObject controller = Instantiate(controllerModel);
                     GameObject parent = new GameObject();
+                    // TODO: Replace this with Controller[Handedness], when possible.
+                    parent.name = "Controller" + Guid.NewGuid().ToString();
+
+                    parent.transform.parent = transform;
```

Comment:
Handedness is in the 16232 SDK which was flighted a few days back. Unity may need to update for us to see it.